### PR TITLE
Clear the meaning of  frequency in step-68

### PR DESCRIPTION
**Pull request by blaisb**
As noted by @tamiko . Step-68 uses confusing language when talking about the number of iteration between the writing of each output files. What we called "frequency" is actually an inverse frequency. The higher it is, the less files we write.
This PR clears this typo and makes things a bit clearer in that regard.

### CODE REVIEW

File: examples/step-68/parameters.prm [hunk 1]
```diff
@@ -9,13 +9,13 @@ subsection Particle Tracking Problem
   set Output directory              = ./
 
   # Iteration frequency at which output results are written
-  set Output frequency              = 10
+  set Output inverse frequency              = 10
```
Comment (bangerth) [inline examples/step-68/parameters.prm:12]: Just call it "Output interval" or "Time steps between output" or some such.

**Review by bangerth — APPROVED** [review]
A frequency is an inverse time interval. You're just making it more complicated than necessary if you introduce an inverse frequency :-)

**Review by bangerth — CHANGES_REQUESTED** [review]
Didn't mean to approve...

File: examples/step-68/parameters.prm (hunk 1 above)
Comment (blaisb) [reply examples/step-68/parameters.prm:12]: Yeah that's right, output interval is fine and dandy :). I'll use that instead.

**Review by tamiko — APPROVED** [review]

**Review by bangerth — APPROVED** [review]

**Pull request merged**